datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -611,6 +611,61 @@ type Dataset implements EntityWithRelationships & Entity {
     View related properties. Only relevant if subtypes field contains VIEW.
     """
     viewProperties: ViewProperties
+
+    """
+    Experimental API.
+    For fetching extra entities that do not have custom UI code yet
+    """
+    aspects(input: AspectParams): [RawAspect!]
+}
+
+
+"""
+Params to configure what list of aspects should be fetched by the aspects property
+"""
+input AspectParams {
+    autoRender: Boolean
```

Comment:
Is this supposed to be for fetching ONLY those aspects which are defined as autorender?

Reply:
nit: no comment on autoRender field